tech/languages/ruby/ruby-installation.md

```diff
@@ -50,7 +50,7 @@ RUBYPICK=_mri_
 The first step is to install dependencies for Ruby.
 
 ```console
-$ sudo dnf install git-core zlib zlib-devel gcc-c++ patch readline readline-devel libyaml-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison curl sqlite-devel perl-FindBin
+$ sudo dnf install -y gcc rust patch make bzip2 openssl-devel libyaml-devel libffi-devel readline-devel zlib-devel gdbm-devel ncurses-devel
```

Comment:
ruby-build's documentation instructs including ncurses-devel for the Ruby environment: https://github.com/rbenv/ruby-build/wiki#fedora

Reply:
Since later we are using git, we need at least git-core, and those perl deps are recommended in "For Ruby < 3 you'll also need to install the following perl modules:"
https://github.com/rbenv/ruby-build/wiki#fedora

perl-lib might get pulled in by one of those 2, but it is required nonetheless, so specifying it won't hurt.